index.html

```diff
@@ -4,14 +4,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Mobile-Version</title>
+    <title>Mobile-Version and Desktop-Version</title>
     <link rel="stylesheet" href="main.css">
   </head>
   <body>
+    <!--mobile version-->
     <header class="header">
       <a href="#"><h4>PEDRO MARQUES</h4></a>
       <nav><img src="images/menu.svg" alt="Menu"></nav>
     </header>
+    <!--desktop version-->
+    <header>
+      <nav>
+        <ul class="topheader">
+          <li class="active"><a href="#">Portfolio</a></li>
+          <li class="active"><a href="#">About</a></li>
+          <li class="active"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- [ ] Please center this section because it's supposed to be in the middle and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/161076504-660fe593-b3b3-4085-adf3-3111d218015f.png)  | ![image](https://user-images.githubusercontent.com/68796232/161076358-3bdcc297-b9de-4242-b40c-3b7cab0e67f7.png)  |

Reply:
Fixed. All CSS file has been reviewed following your recommendations, eliminating margins, and centralizing all contents. In addition, I have reviewed the mobile version too.